shared/unattended/RHEL-6-perf-series.ks

```diff
@@ -0,0 +1,72 @@
+install
+KVM_TEST_MEDIUM
+text
+poweroff
+lang en_US.UTF-8
+keyboard us
+key --skip
+network --bootproto dhcp
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc Asia/Shanghai
```

Comment:
@luckyh I just noticed that I only answered your second question.  So, let me answer the first one probably: I think it's OK for non-chinese users.  If we find issues, this can by another type of "lint" that we can add to the unattended files (we started with username/passwords on another PR).

Anyway, let me know if this is missing something to be approved.  Thanks.